### ISSUE TITLE
Installing a fresh AZERO testnet node issues

### ISSUE THREAD

**Issue by powerpsy** [Cardinal-Cryptography/aleph-node-runner#66]
When installing a new fresh node 13.1 on a new Ubuntu 22.04 server with docker configured as non root,

```
git clone https://github.com/Cardinal-Cryptography/aleph-node-runner
cd aleph-node-runner
```

You get the directory and script `run_node.sh` that will configure the node using the file in validator_testnet

1. For any reason, if you Ctrl+C during the snapshot phase, the script will not perform well at the end (I tried it twice and each time you end it with a restarting container)
2. For any reason, if you didn't correctly configure the `env/validator_testnet` BEFORE launching the script, the container will not be corrected. Example to reproduce:
* start the `run_node.sh` script until the end
* modify ports in the `env/validator_testnet`
* `docker stop` container
* start the `run_node.sh` script again
* You will get a container without the ports you changed. I tried this twice also.

In my case, I wrote "for any reason", to me the reason was that I was using a ssh session, I made the script via the ssh session whereas I wanted to start it directly from the server itself, so I Ctrl+C in the middle of the script from ssh before relaunching it directly on the server.

**Comment by piotrMocz** [comment]
Good catch! Admittedly, we're not handling the interruptions during snapshot download very gracefully right now, will work on that
